src/SolverInterface/DspOsiCpx.h

```diff
@@ -207,6 +207,14 @@ class DspOsiCpx : public DspOsi {
 		CPXsetdblparam(cpx_->getEnvironmentPtr(), CPX_PARAM_EPGAP, CoinMax(0.0, CoinMin(1.0, tol)));
 	}
 
+	/** return unbounded ray */
+	virtual std::vector< double * > getUnbdRay() {
```

Comment:
Should we switch to LP here?

Reply:
add a description of initial solve: CPLEX Error  1254